<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pep8Dialog</class>
 <widget class="QDialog" name="Pep8Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PEP 8 Check Result</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;PEP 8 Check Results&lt;/b&gt;
&lt;p&gt;This dialog shows the results of the PEP 8 check. Double clicking an
entry will open an editor window and position the cursor at the respective line and position.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="filterFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Exclude Files:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="excludeFilesEdit">
        <property name="toolTip">
         <string>Enter filename patterns of files to be excluded separated by a comma</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="clearButton">
        <property name="toolTip">
         <string>Press to clear the filter edit</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="4">
       <widget class="Line" name="line">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="4">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="toolTip">
           <string>Press to start the PEP 8 check run</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loadDefaultButton">
          <property name="toolTip">
           <string>Press to load the default values</string>
          </property>
          <property name="text">
           <string>Load Defaults</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="storeDefaultButton">
          <property name="toolTip">
           <string>Press to store the current values as defaults</string>
          </property>
          <property name="text">
           <string>Store Defaults</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Exclude Messages:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="excludeMessagesEdit">
        <property name="toolTip">
         <string>Enter message codes or categories to be excluded separated by a comma</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="clearButtonExcludeMessages">
        <property name="toolTip">
         <string>Press to clear the exclude messages edit</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="excludeMessagesSelectButton">
        <property name="toolTip">
         <string>Press to select the messages from a list</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Included Messages:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="includeMessagesEdit">
        <property name="toolTip">
         <string>Enter message codes or categories to be included separated by a comma</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="clearButtonIncludeMessages">
        <property name="toolTip">
         <string>Press to clear the include messages edit</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="includeMessagesSelectButton">
        <property name="toolTip">
         <string>Press to select the messages from a list</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QCheckBox" name="repeatCheckBox">
        <property name="toolTip">
         <string>Select to repeat each message type</string>
        </property>
        <property name="text">
         <string>Repeat messages</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="resultList">
     <property name="whatsThis">
      <string>&lt;b&gt;Result List&lt;/b&gt;
&lt;p&gt;This list shows the results of the PEP 8 check. Double clicking
an entry will open this entry in an editor window and position the cursor at
the respective line and position.&lt;/p&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>File/Line</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="checkProgress">
     <property name="toolTip">
      <string>Shows the progress of the PEP 8 check action</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>loadDefaultButton</tabstop>
  <tabstop>excludeFilesEdit</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>excludeMessagesEdit</tabstop>
  <tabstop>clearButtonExcludeMessages</tabstop>
  <tabstop>excludeMessagesSelectButton</tabstop>
  <tabstop>includeMessagesEdit</tabstop>
  <tabstop>clearButtonIncludeMessages</tabstop>
  <tabstop>includeMessagesSelectButton</tabstop>
  <tabstop>repeatCheckBox</tabstop>
  <tabstop>storeDefaultButton</tabstop>
  <tabstop>resultList</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>excludeFilesEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButtonExcludeMessages</sender>
   <signal>clicked()</signal>
   <receiver>excludeMessagesEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButtonIncludeMessages</sender>
   <signal>clicked()</signal>
   <receiver>includeMessagesEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
